markdown/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "markdown"
+version = "0.1.0"
+edition = "2021"
```

Comment:
I'm guessing I should add a `description` and  `license` field. Anything else I should add?